### DISCUSSION TITLE
[WARN] ** [rspamd] Clamav Host 'clamav' is not accessible, retrying..

### DISCUSSION THREAD

**Discussion by Brovning** [tiredofit/docker-rspamd#8] [Q&A]
Hello,

first of all: Thank you very much for this docker container!

I'm trying to use this docker on my Synology DS.
During startup I'm receiving the error message: "[WARN] ** [rspamd] Clamav Host 'clamav' is not accessible, retrying.."

What is the root cause of this error message and how can I solve this issue?

Best regards,

Brovning

**Comment by Brovning** [comment]
Hello @tiredofit ,

do you have any idea what might be the issue?

**Reply by darac** [reply]
Hello,

I'm not @tiredofit, but I'd suggest double checking the following:
* Do you actually have a host named `clamav`? This might be:
  * A container named `clamav`
  * A host on your local network named `clamav`
  If not, then you will need to set the environment variable to tell rspamd where your clamav-daemon is.
* Are you using Docker Swarm? If so, your clamav container might be named `tasks.clamav` instead (Swarm tends to give each instance of a container a random name, but `tasks.clamav` reliably points to the set of instances).
* The rspamd container might not have the same DNS search domain as your other hosts, so you might need to say that your clamav server is located at `clamav.example.com` or perhaps use an IP address.
